Gemfile.lock

```diff
@@ -279,37 +292,39 @@ GEM
       rack (>= 1.0, < 3)
     rack-proxy (0.6.5)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rails (5.2.2.1)
-      actioncable (= 5.2.2.1)
-      actionmailer (= 5.2.2.1)
-      actionpack (= 5.2.2.1)
-      actionview (= 5.2.2.1)
-      activejob (= 5.2.2.1)
-      activemodel (= 5.2.2.1)
-      activerecord (= 5.2.2.1)
-      activestorage (= 5.2.2.1)
-      activesupport (= 5.2.2.1)
+    rack-test (1.1.0)
```

Comment:
check on this

Reply:
this is fine, just involves a new required argument for non-get requests that shouldn't affect us much per https://github.com/rack-test/rack-test/releases/tag/v1.0.0